tests/deepsparse/loggers/test_helpers.py

```diff
@@ -30,5 +42,22 @@
         ("re:string_*..*.string.*", "string_3.string_4", (True, None)),
     ],
 )
-def test_match(template, identifier, expected_output):
+def test_check_identifier_match(template, identifier, expected_output):
     assert check_identifier_match(template, identifier) == expected_output
+
+
+@pytest.mark.parametrize(
```

Comment:
need to test multiple repeated indices, indices between parameters, etc. not sure that these are currently covered
i.e. `key_1[0][0].key2[1].key3`